Try to detect bind mounts (#14831)
netdata/netdata: collectors/diskspace.plugin/README.md

By default, Netdata will enable monitoring metrics only when they are not zero. If they are constantly zero they are ignored. Metrics that will start having values, after Netdata is started, will be detected and charts will be automatically added to the dashboard (a refresh of the dashboard is needed for them to appear though).

> Netdata will try to detect mounts that are duplicates (i.e. from the same device), or binds, and will not display charts for them since most likely the device is already monitored. If you find that some devices are missing, switch `ignore duplicate devices and binds` to `no`.

Reviewer: ```suggestion
Netdata will try to detect mounts that are duplicates (i.e. from the same device), or binds, and will not display charts for them, as the device is usually already monitored. If you find that some devices are missing, set `ignore duplicate devices and binds` to `no`.
```
